JoyboyCommunity/src/components/typography/index.tsx

```diff
@@ -21,6 +21,10 @@ interface TypographyProps extends IStyledTypographyType {
   children: React.ReactNode;
   style?: CSSProperties;
 }
-export default function Typography(props: TypographyProps) {
-  return <StyledTypography {...props}>{props.children}</StyledTypography>;
-}
+
+export const Typography: React.FC<TypographyProps> = ({
```

Comment:
im not sure why this is changed. Was this a wrong implementation

Reply:
Codebase mainly used arrow function expression, so thought all components should be in-sync and use arrow function expressions instead of function declaration.